var/spack/repos/builtin/packages/py-kubernetes/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyKubernetes(PythonPackage):
+    """Official Python client library for kubernetes. """
+
+    homepage = "https://kubernetes.io"
+    git      = "https://github.com/kubernetes-client/python.git"
+    pypi = "kubernetes/kubernetes-17.17.0.tar.gz"
+
+    maintainers = ['vvolkl']
+
+    version('18.17.0a1', sha256='5746f5aeacdd2bd7e5ec2250db2f3ca4008a0c4f26e1b6e444e7f3d663f3af6a')
+    version('17.17.0b1', sha256='2a5cd1b80f821e7b1b4e120bb306fc0ab6e779450df2554f0d8199fcb389f63b')
```

Comment:
Is there any reason to add alpha and beta versions? If not, I would remove them. If you need them, you should mark 17.17.0 as preferred=True.

Reply:
No, not really. I removed them.